components/GetProjects.js

```diff
@@ -9,18 +9,21 @@ const GetProjects = () => {
       <section className="flex-1 text-center text-white">
         <h1 className="mb-5 text-4xl tracking-widest uppercase">Projects</h1>
         <p className="text-white/70">Projects by 4c community Members</p>
-        <div className="flex flex-wrap items-center justify-center pt-4 pb-4 m-2 overflow-y-auto text-left gap-20">
+        <div className="flex flex-wrap items-center justify-center pt-4 my-[1rem] overflow-y-auto text-left gap-20">
```

Comment:
maybe its better if you reduce the margin, cz its super gap in mobile device
![image](https://user-images.githubusercontent.com/88339569/201407987-c3467c3e-1b92-4935-8d54-76ddfc229cba.png)